java/jdbc/src/main/java/com/flipkart/vitess/jdbc/VitessVTGateManager.java

```diff
@@ -78,26 +78,14 @@ private static String getIdentifer(String hostname, int port, String userIdentif
      * @param hostname
      * @param port
      * @param username
-     * @return
-     */
-    private static VTGateConn getVtGateConn(String hostname, int port, String username) {
```

Comment:
This method is dead code... `private` to this class, but not used within this class anymore.  If there's actually a legit reason why it's still lingering around, then I can restore it.